server/app/views/questiontypes/AddressQuestionFragment.html

```diff
@@ -0,0 +1,137 @@
+<div
+  th:fragment="address (question, stateAbbreviations)"
+  th:with="questionId=${#strings.randomAlphanumeric(8)}"
+  th:id="${questionId}"
+>
+  <fieldset
+    class="usa-fieldset"
+    th:with="addressQuestion=${question.createAddressQuestion()}"
+  >
+    <!-- Title and Help Text -->
+    <div
+      th:replace="~{questiontypes/QuestionBaseFragment :: titleAndHelpTextMultipleInput(${question}, ${questionId})}"
+    ></div>
+
+    <!-- Display errors for the entire question -->
+    <div
+      th:replace="~{questiontypes/QuestionBaseFragment :: validationErrors(${addressQuestion.getValidationErrors().get(question.getContextualizedPath())})}"
+    ></div>
+
+    <!-- TODO(#7007): Use ErrorDisplayMode to determine whether to show field errors. -->
+    <!-- Street address -->
+    <th:block th:with="inputId=${#strings.randomAlphanumeric(8)}">
+      <label
+        class="usa-label"
+        th:for="${inputId}"
+        th:text="#{label.street}"
+      ></label>
+      <input
+        class="usa-input"
+        th:id="${inputId}"
+        th:name="${addressQuestion.getStreetPath()}"
+        th:value="${addressQuestion.getStreetValue().orElse('')}"
+        th:aria-invalid="${!addressQuestion.getValidationErrors().isEmpty()}"
+        th:aria-required="${!question.isOptional()}"
+        autocomplete="address-line1"
+      />
+      <div
+        th:replace="~{questiontypes/QuestionBaseFragment :: validationErrors(${addressQuestion.getValidationErrors().get(addressQuestion.getStreetPath())})}"
+      ></div>
+    </th:block>
+
+    <!-- Street address line 2 -->
+    <th:block th:with="inputId=${#strings.randomAlphanumeric(8)}">
```

Comment:
**suggestion (non-blocking):** These all seem to have a similar structure - you could consider creating a AddressTextInputFragment or something that takes the labelText, inputName, inputValue and autocompleteType. If you think this ends up being clearer, that's fine though.

Reply:
These will get changed some with the addition of more CSS classes, but It's something to consider (probably more broadly than just address) to have the equivalent fragment of FieldWithLabel. I'll just have to think about how to handle optional parameters. I'll look into it as a follow up refactor